cf_units/__init__.py

```diff
@@ -1849,10 +1849,10 @@ def __ne__(self, other):
         """
         return not self == other
 
-    def convert(self, value, other, ctype=FLOAT64):
+    def convert(self, value, other, ctype=FLOAT64, inplace=True):
```

Comment:
I agree. It also changes the current behaviour without warning. If this is `False` by default then I think it is OK.